### DISCUSSION TITLE
Renovate doesn't doesn't match v3 to v3.0.0 for release notes

### DISCUSSION THREAD

**Discussion by laurent22** [renovatebot/renovate#26079] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

When creating a pull request, Renovate sometimes populates the "Release Notes", but sometimes doesn't. I'm not entirely sure why.

For example, this pull request:

https://github.com/laurent22/joplin/pull/9428

It has a "Release notes" section but it's empty. Clicking on "v3" also doesn't bring to the GitHub repository release page.

However that release page does exist:

https://github.com/docker/login-action/releases/tag/v3.0.0

Any idea why it's doing this? I assume it's some kind of bug, but at least as a workaround it would be good to put a link to the package release notes.

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment]
expected behavior, not a bug

**Reply by rarkins** [reply]
I think we can get to do better though, eg using semver coerce

**Reply by viceice** [reply]
sounds good but can be hard to implement 🤔
